src/pe/header.rs

```diff
@@ -835,6 +902,139 @@ impl ctx::TryIntoCtx<scroll::Endian> for Header {
     }
 }
 
+/// The DANS marker is a XOR-decoded version of the string "DanS" and is used to identify the Rich header.
+pub const DANS_MARKER: u32 = 0x536E6144;
+/// The Rich marker is a XOR-decoded version of the string "Rich" and is used to identify the Rich header.
+pub const RICH_MARKER: u32 = 0x68636952;
+
+/// The Rich header is a undocumented header that is used to store information about the build environment.
+///
+/// The Rich Header first appeared in Visual Studio 6.0 and contains: a product identifier, build number, and the number of times it was used during the build process.
+#[derive(Debug, PartialEq, Clone, Default)]
+pub struct RichHeader {
+    /// The Rich header data without the padding.
+    pub data: Vec<u8>,
```

Comment:
Sometimes this is useful for calculating hash of rich data, so it would be nice to expose it to consumers. Anyways I am reverting back to byte slices and impl iterator for rich metadatas, as well as how unwind table is implemented.